treeherder/intermittents_commenter/constants.py

```diff
@@ -0,0 +1,100 @@
+import logging
+
+from django.conf import settings
+
+logger = logging.getLogger(__name__)
+TREEHERDER_USER_AGENT = 'treeherder/{}'.format(settings.SITE_HOSTNAME)
+REQUESTS_TIMEOUT = 30
+
+# Min required failures per bug in order to post a comment
+MIN_DAILY_THRESHOLD = 15
+MIN_WEEKLY_THRESHOLD = 1
+TOP_BUGS_THRESHOLD = 50
+
+# Includes call-to-action message for priority threshold bugs
+PRIORITY1_THRESHOLD = 75
+PRIORITY2_THRESHOLD = 30
+
+DISABLE_THRESHOLD = 150
+DISABLE_DAYS = 21
```

Comment:
I think some of these that are only used once would be better inlined right before point of use (still keeping the variable name, if it aids with understanding). This saves having to jump between files and/or add comments that just duplicate the value of the constant - eg:
https://github.com/sarah-clements/treeherder/blob/b652af0f1635989475038d426ff211a00130aad2/treeherder/intermittents_commenter/commenter.py#L63-L64

Reply:
That ends up being most of those values... a little embarrassed I didn't think of this myself.